Sources/Sentry/SentryANRTrackingIntegration.m

```diff
@@ -142,15 +126,17 @@ - (void)anrDetectedWithType:(enum SentryANRType)type
         return;
     }
 
-    NSString *message = [NSString stringWithFormat:@"App hanging for at least %li ms.",
-        (long)(self.options.appHangTimeoutInterval * 1000)];
+    NSString *appHangDurationInfo = [NSString
+        stringWithFormat:@"at least %li ms", (long)(self.options.appHangTimeoutInterval * 1000)];
+    NSString *message = [NSString stringWithFormat:@"App hanging for %@.", appHangDurationInfo];
     SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
 
     NSString *exceptionType = [SentryAppHangTypeMapper getExceptionTypeWithAnrType:type];
     SentryException *sentryException = [[SentryException alloc] initWithValue:message
                                                                          type:exceptionType];
 
-    sentryException.mechanism = [[SentryMechanism alloc] initWithType:@"AppHang"];
+    SentryMechanism *mechanism = [[SentryMechanism alloc] initWithType:@"AppHang"];
```

Comment:
`l`: Is there a value in having a list of constants/"enum-like" for the type?

Reply:
Nope, good point. I will add one after merging this PR.